apps/docs/public/manifest.webmanifest

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Puck",
+  "short_name": "Pck",
```

Comment:
Interesting. Does this have to be 3 characters?

Reply:
Nope, it doesn't have to be anything. Could just be Puck again I guess

Reply:
It's only used if there is not enough space to display the `name`, so probably moot in the case of "Puck"

https://developer.mozilla.org/en-US/docs/Web/Manifest/short_name